scan/src/pages/BlockIndexPage.re

```diff
@@ -109,7 +103,7 @@ let make = (~height: int) => {
     <Row>
       <Col size=1.8>
         {switch (blockOpt) {
-         | Some(block) => <InfoHL info={InfoHL.Count(block.numTxs)} header="TRANSACTIONS" />
+         | Some(block) => <InfoHL info={InfoHL.Count(block.txn)} header="TRANSACTIONS" />
```

Comment:
I thought we have many ways to handle None/NoData. Now I think we use let%Sub in top of make function and return Sub.resolve. Look example at DataSourcePage.re

Reply:
Waiting for TxSub